reg/arginfo.py

```diff
@@ -93,18 +94,15 @@ class Dummy(object):
 
 
 def get_class_init(class_):
-    try:
-        func = class_.__init__
-    except AttributeError:  # pragma: no cover
-        # Python 2 classic class without __init__.
-        return fake_empty_init
+    func = class_.__init__
+
     # If this is a new-style class and there is no __init__
```

Comment:
I was hoping that the differences between PyPy 3 and Python 3.7 were less, but the code to handle this is still in place. Does this mean this is still needed?

Reply:
I would like us to look into this before we merge it. But if the tests do work we can instead choose to merge it anyway and look into it later (create a github issue for Reg if you decide to do that). So I'm approved in principle.

Reply:
@faassen I have removed the PyPy specific code. Also removed some PyPy code in Importscan (importscan/scan.py).